### PR TITLE
NPC with camera skin

### LINKED ISSUE #209: ENH: Put an NPC with camera skin in camera location
**Is your feature request related to a problem? Please describe.**
Right now it is impossible for prisioners to know where the guards have placed cameras.

**Describe the solution you'd like**
When a camera is placed it should be represented as a NPC with a camera skin. This NPC would replace the armorstand.

### PR DESCRIPTION
**Pull request by TiagoFar78**
Closes #209

### CODE REVIEW

File: TestServer/plugins/TF_PrisonEscape/config.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-PrisionersRatio: 0.73
-PoliceRatio: 0.27
+PrisionersRatio: 0.27
+PoliceRatio: 0.73
```
Comment (iquelli) [inline TestServer/plugins/TF_PrisonEscape/config.yml:2]: Correct the ratios

**Review by iquelli — CHANGES_REQUESTED** [review]

File: src/main/resources/config.yml [hunk 2]
```diff
@@ -37,6 +37,9 @@ AvailableLanguages:
 - "english"
 DefaultLanguage: "english"
 TeamChatPrefix: "t "
+# Get signature and texture from here: https://mineskin.org/gallery?query=Camera
```
Comment (iquelli) [inline src/main/resources/config.yml:40]: Remove comment, but it's just a suggestion. Up to you

Comment (TiagoFar78) [reply src/main/resources/config.yml:40]: I had to do a lot of research to know how to work with skins, I think it's really useful to have that there so you can understand fast how do change the skin.

Comment (iquelli) [reply src/main/resources/config.yml:40]: Fair :)

**Review by iquelli — APPROVED** [review]

**Pull request merged**